workspace.json

```diff
@@ -1700,6 +1700,35 @@
         }
       }
     },
+    "application-templates-institution-application": {
+      "root": "libs/application/templates/institution-application",
+      "sourceRoot": "libs/application/templates/institution-application/src",
+      "projectType": "library",
+      "schematics": {},
+      "architect": {
+        "lint": {
+          "builder": "@nrwl/linter:lint",
+          "options": {
+            "linter": "eslint",
+            "tsConfig": [
+              "libs/application/templates/institution-application/tsconfig.lib.json",
+              "libs/application/templates/institution-application/tsconfig.spec.json"
+            ],
+            "exclude": [
+              "**/node_modules/**",
+              "!libs/application/templates/institution-application/**/*"
+            ]
+          }
+        },
+        "test": {
+          "builder": "@nrwl/jest:jest",
+          "options": {
+            "jestConfig": "libs/application/templates/institution-application/jest.config.js",
+            "passWithNoTests": true
+          }
+        }
+      }
```

Comment:
You need to define the extract strings script to handle translation with contentful https://docs.devland.is/handbook/libs/localization#message-extraction